### ISSUE TITLE
add: tmate and create ssh key automatically

### ISSUE THREAD

**Issue by steadfasterX** [Carbon-Fusion/build_fwul#60]
### Description

add **tmate** which is great if the X server is not available or when installing teamviewer is too much/not needed.
tmate is fast, small and works over slow connections.
TeamViewer consumes more space (thats why -beside license- it isn't directly integrated into FWUL) , the download is big and the whole installation takes a while due to its dependencies and the installation process of the whole thing.

another feature and this is where tmate depends on is: 
automatic generation of a ssh key. This has to be done on boot/after login (including a precheck if its necessary i.e. on persistent installs) as I will not ship the same ssh key for everyone with FWUL.

TLDR;
Integrating tmate into FWUL would help users with low bandwidth internet connections and when no graphical desktop is available

### Links

https://tmate.io/

**Comment by steadfasterX** [comment]
Requires ssh-keygen!! 

yaourt -S tmate
